osmosis/assetlist.json

```diff
@@ -27510,14 +27510,14 @@
           "exponent": 0
         },
         {
-          "denom": "labs",
+          "denom": "LABS",
           "exponent": 6
         }
       ],
       "type_asset": "sdk.coin",
       "base": "factory/osmo1are7fpe5l6jzm9sjn7u4qkq6q77wwrrsxzlyw8lcegmmaxdukvuq4h46dx/LABS",
       "name": "Labrador",
-      "display": "labs",
+      "display": "LABS",
```

Comment:
note that changing this doesn't really do anything. Apps use Symbol and Name but the "display" property is only for deriving the preferred exponent representation